frontend/apps/web/components/RunTImeline/RunTimeline.tsx

```diff
@@ -465,6 +571,8 @@ function getTaskStatus(
       case 'ActivityTaskCancelRequested':
         isCanceled = true;
         break;
+      case 'ActivityTaskScheduled':
```

Comment:
this switch statement does not include `ActivityTaskStarted` which is mentioned on `L652`

Reply:
originally i was going to say it shouldn't need it because we only want to return a separate status from `running` if its complete or failed or canceled or timedout, but for posterity i'll add it to have coverage